src/sentry/migrations/0384_backfill_installation_ids.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 2.2.28 on 2023-03-09 23:03
+
+from django.db import migrations
+
+from sentry.monitors.models import MonitorStatus
```

Comment:
We generally prefer to not import things from external places - it can cause migrations to break later. Better to just copy that enum in here.